plugins/modules/zos_fetch.py

```diff
@@ -170,66 +171,66 @@
 
 RETURN = r"""
 file:
-    description: The source file path on remote machine
+    description: The source file path on the remote machine.
```

Comment:
I believe this should read:
"The source file path  or data set on the remote machine."

`src` according to the options can be a UNIX file or data set.